uPortal-api/uPortal-api-rest/src/test/java/org/apereo/portal/rest/layout/TabListOfNodesTest.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to Apereo under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright ownership. Apereo
+ * licenses this file to you under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the License at the
+ * following location:
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apereo.portal.rest.layout;
+
+
+import java.io.StringReader;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import junit.framework.Assert;
```

Comment:
`junit.framework.Assert` is deprecated, prefer `org.junit.Assert` instead. 

ref: https://stackoverflow.com/questions/24013210/assert-in-junit-framework-has-been-deprecated-what-use-next